pylib/System/TestDef.py

```diff
@@ -257,19 +258,17 @@ def loadPlugins(self, basedir, topdir):
                 # class definition
                 plugindirs.insert(0, y)
 
-        # Traverse the plugin directory tree and add all
-        # the class definitions we can find
+        # Load plugins from each of the specified plugin dirs
         for dirPath in plugindirs:
             try:
-                filez = os.listdir(dirPath)
-                for file in filez:
-                    file = os.path.join(dirPath, file)
-                    if os.path.isdir(file):
-                        self.loader.load(file)
-            except:
-                if not self.options['ignoreloadpatherrs']:
-                    print("Plugin directory",dirPath,"not found")
-                    sys.exit(1)
+                if not Path(dirPath).exists():
```

Comment:
Minor quibble: will `Path(dirPath).exists()` ever throw an exception?  If not, does it belong inside this `try` block?  More specifically: is the `self.loader().load(...)` the only thing that this `try` block is really protecting?